guides/common/modules/proc_adding-custom-deb-repositories.adoc

```diff
@@ -10,8 +10,10 @@ To use the CLI instead of the {ProjectWebUI}, see the xref:cli-adding-custom-deb
 {Project} automatically completes the *Label* field based on what you have entered for *Name*.
 . Optional: In the *Description* field, enter a description for the repository.
 . From the *Type* list, select `deb` as type of repository.
-. In the *Upstream URL* field, enter the URL of the external repository to use as a source.
+. Optional: In the *Upstream URL* field, enter the URL of the external repository to use as a source.
 You can find the upstream URLs on Debian-based systems in `/etc/apt/sources.list`.
++
+If you do not enter a upstream URL, you can manually upload packages.
```

Comment:
**an** upstream URL (it goes by pronunciation, not spelling) -- also below

Reply:
Fixed both times.